control/nlsys.py

```diff
@@ -1327,8 +1327,8 @@ def input_output_response(
 
     Parameters
     ----------
-    sys : InputOutputSystem
-        Input/output system to simulate.
+    sysdata : I/O system or list of I/O systems
```

Comment:
It is useful to include a class name here because, in that case, docs generation results in a link to documentation for the class. Unless I missed something, this should be `NonlinearIOSystem`. This will work even when the type is written as "...or list of...", e.g., compare with the `data` parameter of `nyquist_plot`: <https://python-control.readthedocs.io/en/0.10.0/generated/control.nyquist_plot.html>